components/organisation-unit-tree/src/organisation-unit-tree/organisation-unit-tree.js

```diff
@@ -150,30 +174,23 @@ OrganisationUnitTree.propTypes = {
     /** When provided, the 'isUserDataViewFallback' option will be sent when requesting the org units */
     isUserDataViewFallback: propTypes.bool,
 
-    /** Renders the actual node component for each leaf, can be used to
-     * customize the node. In order to change the displayed node while keeping
-     * the existing functionality intact, you can re-use the original prop
-     * and overwrite the label property.
+    /** Renders the actual node label content for each leaf, can be used to
```

Comment:
JSDocs was replaced with react-docgen in https://github.com/dhis2/ui/pull/688, so this prop's description needs to be updated to:

```
    /** Renders the actual node component for each leaf, can be used to
     * customize the node. The default function just returns the node's
     * displayName
     * 
     * Shape of the object passed to the callback:
     * ```
     * {
     *    node: {
     *      displayName: string,
     *      id: string,
     *      // Only provided once `loading` is false
     *      path?: string,
     *      // Only provided once `loading` is false
     *      children?: Array.<{
     *        id: string,
     *        path: string,
     *        displayName: string
     *      }>
     *    },
     *    loading: boolean,
     *    error: string,
     *    open: boolean,
     *    selected: string[],
     *    singleSelection: boolean,
     *    disableSelection: boolean,
     *    // If the request is being customized, then all responses except the org
     *    // unit's response data will be included in this object
     *    additional?: object
     * }
     * ``` */
    renderNodeLabel: propTypes.func,
```